README.md

```diff
@@ -37,6 +30,34 @@ Add this to your maven project **root** pom.xml :
     </plugin>
   </plugins>
 </build>
+
+<profiles>
+  <profile>
+    <!-- Install pre-commit hook if missing-->
+    <id>install-pre-commit-hook</id>
+    <activation>
+      <file>
+        <missing>.git/hooks/pre-commit</missing>
+      </file>
+    </activation>
+    <build>
+      <plugins>
+        <plugin>
+          <groupId>com.cosium.code</groupId>
+          <artifactId>maven-git-code-format</artifactId>
+          <executions>
+            <execution>
+              <id>install-hook</id>
+              <goals>
+                <goal>install-hooks</goal>
+              </goals>
+            </execution>
+          </executions>
+        </plugin>
+      </plugins>
+    </build>
+  </profile>
+</profiles>
 ```
```

Comment:
I don't think such "advanced" configuration should be part of the quickstart.
Please revert it or move it to a new section.